.drone.star

```diff
@@ -75,6 +75,14 @@ def main(ctx):
             trigger = cron_trigger,
         ),
         gui_tests(ctx, trigger = cron_trigger),
+        notification(
+            name = "build",
+            trigger = cron_trigger,
+            depends_on = [
+                "clang-debug-ninja",
+                "GUI-tests",
```

Comment:
Note that the notificatioin of nightly depends upon `GUI-tests` and `clang-debug-ninja`. If anyone thinks there should be more then please suggest.